app/src/main/java/org/mozilla/fenix/components/metrics/Metrics.kt

```diff
@@ -245,16 +253,17 @@ sealed class Event {
             SAVE_TO_COLLECTION
         }
 
-        override val extras: Map<String, String>?
-            get() = mapOf("item" to item.toString().toLowerCase())
+        override val extras: Map<Events.browserMenuActionKeys, String>?
+            get() = mapOf(Events.browserMenuActionKeys.item to item.toString().toLowerCase())
     }
 
     sealed class Search
 
-    open val extras: Map<String, String>?
+    internal open val extras: Map<*, String>?
         get() = null
 }
 
+// TODO: Can we get rid of string mapping here?
```

Comment:
Same for string mapping here :)

Reply:
This is pattern matching the `Fact` which is from A-C converting it to an Event. So no need to change anything here. Tests wouldn't hurt though :D